.gitignore

```diff
@@ -20,7 +20,7 @@ pids
 *.seed
 
 # Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
+/lib-cov
```

Comment:
Maybe have all the directories listed in the .gitignore have slashes for consistency purposes...

Reply:
This seems like a pointless change... and it would be silly to change every other line to match this one when they are following a pretty common style pattern that works.